api/ccapi/include/layer.h

```diff
@@ -106,6 +107,7 @@ class Layer {
    * - out_dim : int ( output dimesion for embedding layer )
    * - in_length : int ( input length for embedding layer )
    * - recurrent_activation : string (type) - used only in lstm
+   * - dist_layer : string (type) - layer name to be distributed
```

Comment:
intended?

Reply:
I will remove. it accidentally there. my bad.